app/src/main/java/helium314/keyboard/keyboard/KeyboardSwitcher.java

```diff
@@ -464,6 +471,22 @@ public void switchOneHandedMode() {
         Settings.getInstance().writeOneHandedModeGravity(mKeyboardViewWrapper.getOneHandedGravity());
     }
 
+    // Displays a toast-like message with the provided text resource ID for a specified duration.
+    public void showFakeToast(final int resId, final int timeMillis) {
+        if (isToastShowing) return;
+
+        mFakeToastView.setText(resId);
+        mFakeToastView.setVisibility(View.VISIBLE);
+        isToastShowing = true;
+        mFakeToastView.startAnimation(AnimationUtils.loadAnimation(mLatinIME, R.anim.fade_in));
+
+        new Handler().postDelayed(() -> {
```

Comment:
Android Studio shows "_'Handler' is abstract; cannot be instantiated_".
Why don't you just do `mFakeToastView.postDelayed`?

Reply:
Weird mine did not show such a message
I did not know that views also have postDelay, thank you for telling me.
The handler will be removed